dashboard/app/models/pd/fit_weekend1819_registration.rb

```diff
@@ -25,16 +25,11 @@ class Pd::FitWeekend1819Registration < ApplicationRecord
   belongs_to :pd_application, class_name: 'Pd::Application::ApplicationBase', optional: true
 
   after_create :update_application_status
-  after_create :send_fit_weekend_confirmation_email
 
   def update_application_status
```

Comment:
If the controller for the weekend registration is being deleted, what is the need for keeping the rest of these files?

Reply:
Sorry I probably should've specified, the "these files" I'm referring to are the files you modified but didn't delete

Reply:
I posted here https://codedotorg.slack.com/archives/C0T10HG6N/p1670258684933639 about deleting the model files. I'm not sure it's kosher to delete the model files when we still want to keep the table around –– so for now, I'm keeping the models until I learn what the best practice is

Reply:
Sounds good! Would it be possible to add the comment in those files that Ben suggested just for future reference on why the files are "gutted"? This is just a comment though so maybe not worth a full rebuild, totally your call!

Reply:
Good call! I'm going to see if it's possible to remove all of the model files. If it is, II won't add anything here. If it's not, I'll remove everything from the model files other than the annotations with a comment explaining the dealio